README.md

```diff
@@ -19,11 +19,18 @@ $ cat /etc/gu/explainer.stage.conf
 stage=DEV
 ```
 
+Configure the composer route in nginx:
+
+```
+cd <path_of_dev_nginx>
+sudo ./setup-app.rb <path_of_explainer>/nginx/nginx-mapping.yml
```

Comment:
Yes :)
But for some reason many of the other Editorial Tools READMEs run that command from within the nginx folder. I just let my self contaminated by the existing conventions :)